### DISCUSSION TITLE
Formulários de contato

### DISCUSSION THREAD

**Discussion by rberts** [frontendbr/forum#140] [Discussão]
E aí pessoas! Não sei se já rolou alguma issue sobre esse assunto, mas queria saber o que vocês geralmente usam para formulários de contato em sites estáticos da vida (jekyll, hexo, middleman e outros). Se existe algum app para integrar ou algum projeto open-source com esse tipo de coisa, ou cada um faz na mão um php mesmo pra enviar? Eu vi que muitos sites geralmente nem usam formulários, só coloca um link mailto (eu particularmente não vejo muita necessidade também dependendo do projeto), mas em casos de cliente, muitas vezes é necessário.

**Comment by lucasmahle** [comment]
Dias atrás, eu achei esse artigo aqui.
https://medium.com/@mariusc23/send-an-email-using-only-javascript-b53319616782#.qlfqd095o

**Comment by lucasmahle** [comment]
Ignora, o Mandril é pago kk.
Da uma olhada no http://www.emailjs.com/

**Comment by lazarofl** [comment]
Esse emailJS é bem interessante, vou testar 👍

**Comment by rberts** [comment]
Po, esse emailjs aí parece ser lindo, haha. Já vou testar ele no próximo projeto! ;)
